src/test/java/api/support/fakes/FakeStorageModule.java

```diff
@@ -618,6 +625,11 @@ private void checkForUnexpectedQueryParameters(RoutingContext routingContext) {
         String.join(",", unexpectedParameters)));
   }
 
+  private boolean isContainsQueryParameter(String queryParameter) {
+    String query = StringUtils.substringBefore(queryParameter, "=");
```

Comment:
Why is there a need to perform a substring on the query parameter?

Reply:
This is needed because after rawQuery.split("&") above we have value for example "session_inactivity_time_limit=2019-10-04T13:00:09.416Z"
and then we need to take substring before "=" to get the name of the query parameter